packages/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -25,6 +25,11 @@ export interface TinyliciousClientProps {
      * Optional. A logger instance to receive diagnostic messages.
      */
     logger?: ITelemetryBaseLogger;
+    /**
+     * Optional. Force all connections to start the connection mode as write instead of read.
+     * Setting to `true` enables write mode by default.
```

Comment:
Maybe you want to call out here that based on permissions you may still connect in read mode.

Reply:
Thinking about this again we probably do not need this configuration option. If we do offer it, then we need to reason and explain the benefits of using it vs not using it. And also explain how it relates to the auth permissions, which becomes complicated. Simpler approach would be then to always initiate connection in "write" mode, and then let the connection resolve to read or write mode.